service/src/main/java/gov/nasa/pds/api/registry/controllers/ProductsController.java

```diff
@@ -367,7 +363,7 @@ private PdsLidVid resolveLatestLidvid(PdsProductIdentifier identifier)
     SearchRequest searchRequest = new RegistrySearchRequestBuilder(this.connectionContext)
             .matchLid(identifier.getLid())
             .fieldsFromStrings(List.of())
-            .onlyLatest()
+            .noSupersededProducts()
```

Comment:
I feel like positive assertion are easier to understand, but I see your idea of being more explicit regarding what this method actually does. No need to revert that.

Reply:
@tloubrieu-jpl I tend to agree. `excludeSupersededProducts()`?